game-core/src/main/java/games/strategy/util/IntegerMap.java

```diff
@@ -173,11 +167,9 @@ public T lowestKey() {
    * @return The sum of all keys.
    */
   public int totalValues() {
-    int sum = 0;
-    for (final int value : mapValues.values()) {
-      sum += value;
-    }
-    return sum;
+    return mapValues.values().stream()
+        .mapToInt(value -> value)
```

Comment:
Does `Function.identity()` work here?

Reply:
sadly does not, incompatible types. I was thinking as well something like that should have worked. My IDE is not giving any simplification suggestions, it usually would in this case. I don't fully understand why it does not, but it seems likely there is a solid reason it does not work and hence no auto-suggestion.

Reply:
Well the most likely reason it does not work, is because the mapToInt method requires a ToIntFunction argument, which doesn't even extend Function.
And the identity function does well return a normal function.